src/helpers/articles/rateArticleHelper.js

```diff
@@ -0,0 +1,49 @@
+import db from '../../sequelize/models';
+
+const { Article } = db;
+
+/**
+ * @exports ArticRatelehelper
+ * @class Articlehelper
+ * @description Helps to generate token and passwor hashing
+ * */
+class ArticleRatelehelper {
+  /**
+       * Check the environment
+       * @function validateArticleRated
+       * @param  {integer} id - Check the rate
+       * @param  {integer} user - Check the user
+       * @return {string} Validate the rate
+       */
+  static async validateArticleRated(id, user) {
+    const findarticle = await Article.findOne({ where: { id } });
+    if (findarticle) {
```

Comment:
use camelCases here `findarticle` should instead be `findArticle`

Reply:
thanks @Malaba6. Done